manifests/init.pp

```diff
@@ -49,10 +50,22 @@
     provider => 'gem',
   }
 
+  if (is_array($scout_roles)) {
+    $scout_roles_config = inline_template('<% @scout_roles.size.to_i %>') ? {
```

Comment:
would using stuff in stdlib be simpler than inline templating? is_array join and so on?